code/modules/jobs/job_types/legion.dm

```diff
@@ -266,7 +266,7 @@ Decan
 Veteran
 */
 /datum/job/f13vetlegion
-	title = "Veteran Legionnaire"
+	title = "Veteran Legionary"
```

Comment:
Legionnaire is in fact correct, as it refers explicitly to ancient roman legionnaires.

Reply:
according to wikipedia and a reenactor's website, legionnaire is in reference to modern day representations of members of a legion, and legionary is a reference to imperial roman legions, on the basis of the translation being different based on time period.